tests/general/ctors_dtors.c.stdout.expected

```diff
@@ -5,6 +5,6 @@ hello main
 goodbye main
 hello another_from_atexit
 hello from_atexit
-hello from_destructor65535
 hello from_destructor
+hello from_destructor65535
```

Comment:
In native code, `from_destructor65535` runs before `from_destructor`, and I believe the existing expected output is correct. And with newer LLVM releases that have [this commit](https://github.com/llvm/llvm-project/commit/d865437d9c82f814dd3b0d0e60042e2c176e201f) in them, that's what wasi-sdk does too.

Assuming that this is the underlying cause of the difference here, would it be practical for you to user a newer LLVM here?